.github/actions/vpn-azure/action.yml

```diff
@@ -71,22 +71,30 @@ runs:
       shell: bash
 
     - name: Update VPN DNS IP
-      if: inputs.dns-ip != 'false' && inputs.ca-cert != 'false'
+      if: inputs.dns-ip != '127.0.0.1' && inputs.ca-cert != 'false'
       run: |
         sed -i "s/\(dhcp-option DNS \).*/\1${{ inputs.dns-ip }}/" .github/vpn/${{ env.VPN_ENV }}.ovpn
       shell: bash
 
-    - uses: josiahsiegel/action-connect-ovpn@a4c76bd3ba523f065468d633a40d16d1175bcb79
+    - uses: josiahsiegel/action-connect-ovpn@794339aff94452216c97f609476c367a43a31295
```

Comment:
dependabot recognizes [josiahsiegel/action-connect-ovpn@e9fb6a005b57431ef6e564c3c67434378a49951f](https://github.com/CDCgov/prime-reportstream/pull/12102).  Is this the release commit?

Reply:
dependabot has trouble with the versioning system changes.  This change is correct and should trigger dependabot to start using the new versioning.